README.md

```diff
@@ -107,6 +107,22 @@ JOBS_USERNAME=jobuser
 JOBS_PASSWORD=jobpassword
 ```
 
-## Usage in tests
+## Publishing New Version
 
-## Contribute
+Create a branch that doesn't have the same name as the tag e.g. v1.0.120
+
+```
+git checkout version-<some_version>
+```
+
+Increase the version
+
+```
+npm version <some_version>
```

Comment:
I usually just do `npm version patch` and it increases version by 1

Reply:
can we update it to `npm version patch` and not provide the number manually?

Reply:
Yes: https://github.com/kontist/mock-solaris/pull/182/commits/ab38ea7298fbc8f87c43cf7c056f204208075beb